src/apis/carts.ts

```
@@ -29,20 +29,28 @@ export const getCartGlassList = async () => {
+        count
       },
-    })
-    return response;
+      {
+        headers: {
+          Authorization: `Bearer ${await getItem('accessToken')}`,
+        },
+      }
+    );
   } catch (err) {
     throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)